upup/pkg/fi/cloudup/populate_instancegroup_spec.go

```diff
@@ -31,14 +31,19 @@ import (
 
 // Default Machine types for various types of instance group machine
 const (
-	defaultNodeMachineTypeAWS = "t2.medium"
-	defaultNodeMachineTypeGCE = "n1-standard-2"
+	defaultNodeMachineTypeAWS     = "t2.medium"
+	defaultNodeMachineTypeGCE     = "n1-standard-2"
+	defaultNodeMachineTypeVSphere = "vsphere_node"
 
-	defaultBastionMachineTypeAWS = "t2.micro"
-	defaultBastionMachineTypeGCE = "f1-micro"
+	defaultBastionMachineTypeAWS     = "t2.micro"
+	defaultBastionMachineTypeGCE     = "f1-micro"
+	defaultBastionMachineTypeVSphere = "vsphere_bastion"
 
-	defaultMasterMachineTypeGCE = "n1-standard-1"
-	defaultMasterMachineTypeAWS = "m3.medium"
+	defaultMasterMachineTypeGCE     = "n1-standard-1"
+	defaultMasterMachineTypeAWS     = "m3.medium"
+	defaultMasterMachineTypeVSphere = "vsphere_master"
+
+	defaultVSphereNodeImage = "ubuntu_16_04"
```

Comment:
We need to fix default image name extraction for vSphere. Here is the relevant issue https://github.com/vmware/kops/issues/50.

We can remove this default image from here, but note that that will make '--image' a mandatory flag for vSphere cluster create operation.